docs/best-practices/gitops.md

```diff
@@ -278,7 +278,7 @@ GitHub Actions pipelines are created and executed:
 
 ### 8.3 ArgoCD Installation
 
-Argo CD is installed in your Kubernetes cluster:
+ArgoCD is installed in your Kubernetes cluster:
```

Comment:
Argo CD is right.

![image](https://user-images.githubusercontent.com/18692628/205895801-17055ab8-f20b-48de-aada-a2f8c8599b98.png)

Please help to change the title here.

Reply:
Done